### PR TITLE
fix(styles): remove tmp variables, old mixins [KHCP-8944]

### PR DESCRIPTION
**Pull request by portikM**
# Summary

Clean up styles and mixins, removing temporary variables and old mixins

## PR Checklist

* [ ] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/kongponents#committing-changes).
* [ ] **Tests coverage:** test coverage was added for new features and bug fixes
* [ ] **Docs:** includes a technically accurate README

### CODE REVIEW

File: src/components/KCodeBlock/KCodeBlock.vue [hunk 1]
```diff
@@ -715,6 +715,10 @@ const getIconColor = computed(() => props.theme === 'light' ? KUI_COLOR_TEXT_NEU
 </script>
 
 <style lang="scss" scoped>
+/* Component variables */
+
+$kCodeBlockDarkLineMatchBackgroundColor: rgba(255, 255, 255, 0.12); // we don't have a token for this
```
Comment (adamdehaven) [inline src/components/KCodeBlock/KCodeBlock.vue:720]: "dark line match background color" - I have no idea what this means, or what it's used for 😬  Could you add a comment or a better name for the var?

File: src/styles/styles.scss [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 :root {
   body {
-    // font weight
+    font-family: var(--kui-font-family-text, $kui-font-family-text);
```
Comment (adamdehaven) [inline src/styles/styles.scss:8]: issue: we don't want to define a font family on the body; this is controlled by the host app

**Review by adamdehaven — CHANGES_REQUESTED** [review]

File: src/components/KCodeBlock/KCodeBlock.vue (hunk 1 above)
Comment (portikM) [reply src/components/KCodeBlock/KCodeBlock.vue:721]: Done

**Review by adamdehaven — APPROVED** [review]
lgtm

**Review by adamdehaven — APPROVED** [review]

**Pull request merged**